### PR TITLE
Unit tests for LocalCsvReader. It was tested using react with jest.Th…

### PR DESCRIPTION
**Pull request by Joembonayo**
Closes #101 

# Fix failed tests

# Implementaion of Unit Tests on LocalCsvReader. Not all test succeed

# It was tested using react with jest

### CODE REVIEW

**Comment by matthew-buglass** [comment]
There is nothing in this PR and the work and PR description is not filled out.

File: my-webxr-app/tests/DraftUnitTests.test.tsx
Comment (matthew-buglass) [inline my-webxr-app/tests/DraftUnitTests.test.tsx:1]: Why is the test file empty and no other work here?

**Review by matthew-buglass — CHANGES_REQUESTED** [review]

**Comment by Joembonayo** [comment]
> There is nothing in this PR and the work and PR description is not filled out.

My unit tests are in "DraftUnitTests.test.tsx". package.json was also updated. Changes are in the feature-branch

File: my-webxr-app/tests/DraftUnitTests.test.tsx
Comment (Joembonayo) [reply my-webxr-app/tests/DraftUnitTests.test.tsx:1]: I honestly don't know why it is empty.

**Comment by Joembonayo** [comment]
> There is nothing in this PR and the work and PR description is not filled out.

The changes are in DraftUnitTests

File: my-webxr-app/tests/DraftUnitTests.test.tsx
Comment (matthew-buglass) [inline my-webxr-app/tests/DraftUnitTests.test.tsx:1]: The file should also be more descriptive. `DraftUnitTests` doesn't tell us anything about what is being tested in the file.

Comment (Joembonayo) [reply my-webxr-app/tests/DraftUnitTests.test.tsx:1]: It is not completely working but I can change the name.

**Pull request closed without merging**